postgres/manage.sls

```diff
@@ -55,16 +73,16 @@ postgres-reload-modules:
   {%- if 'tablespace' in db %}
       - postgres_tablespace: postgres_tablespace-{{ db.tablespace }}
   {%- endif %}
-
 {%- endfor %}
 
 # Schema states
 
 {%- for name, schema in postgres.schemas|dictsort() %}
 
 {{ format_state(name, 'postgres_schema', schema) }}
-  {%- if 'owner' in schema %}
     - require:
+      - postgres_database-{{ dbname }}
```

Comment:
This addition would not work properly, because there are no `dbname` Jinja variable was ever assigned.
You wanted to put `schema.dbname`, right?